flocker/control/test/test_protocol.py

```diff
@@ -50,6 +54,26 @@
 from .clusterstatetools import advance_some, advance_rest
 
 
+def random_ip():
+    return inet_ntop(AF_INET6, urandom(16))
+
+
+def arbitrary_transformation(deployment):
+    """
+    Make some change to a deployment configuration.  Any change.
+
+    The exact change made is unspecified but the resulting ``Deployment`` will
+    be different from the given ``Deployment``.
```

Comment:
It's not specified by the interface.  The interface only specifies that something will change, not what.  Code shouldn't depend on the exact change, only that `deployment != arbitrary_transformation(deployment)`.